loader.js

```diff
@@ -1,6 +1,30 @@
 const gql = require('./');
 
+// Takes `source` (the source GraphQL query string)
+// and `doc` (the parsed GraphQL document) and tacks on
+// the imported definitions.
+function expandImports(source, doc) {
+  const lines = source.split('\n');
+  let outputCode = "";
+  
+  lines.some((line) => {
+    if (line[0] === '#') {
```

Comment:
I think you probably want to check if this comment is an import - so maybe check for `#import`. People should still be able to do regular comments in their queries.